mkdocs-base.yml

```diff
@@ -114,103 +125,154 @@ extra:
+        - release-notes/8.0/8.0.8.md
+        - release-notes/8.0/8.0.7.md
+        - release-notes/8.0/8.0.6.md
+        - release-notes/8.0/8.0.5.md
+      - 2018:
+        - release-notes/8.0/8.0.4.md
+        - release-notes/8.0/8.0-3-rc1.md
+  - Discover Percona XtraBackup:
+      - intro.md
+      - how_xtrabackup_works.md
+      - xtrabackup-version-numbers.md
+  - Features:
+      - Overview: features-overview.md
+      - Tools:
+        - manual.md
+        - The xtrabackup binary:
+          - The xtrabackup binary overview: xtrabackup_bin/xbk_option_reference.md
+          - xtrabackup_bin/analyzing_table_statistics.md
+        - xbcrypt/xbcrypt.md
+        - The xbstream binary: 
+          - The xbstream binary overview: xbstream/xbstream.md
+          - howtos/recipes_xbk_stream.md
+        - The xbcloud binary:
+          - The xbcloud binary overview: xbcloud/xbcloud.md
+          - Use the xbcloud binary:
+            - xbcloud/xbcloud_s3.md
+            - xbcloud/xbcloud-iam-profile.md
+            - xbcloud/xbcloud_swift.md
+            - xbcloud/xbcloud_gcs.md
+            - xbcloud/xbcloud_azure.md
+            - xbcloud/xbcloud_minio.md
+      - xbcloud/xbcloud_exbackoff.md
+      - advanced/log_enhancements.md
+      - advanced/locks.md
+      - advanced/smart_memory_estimation.md
+      - xtrabackup_bin/flush-tables-with-read-lock.md
+      - xtrabackup_bin/working_with_binary_logs.md
+  -  Quick start guide:
+      - quickstart.md
+  - Install:
+      - Before you start:
+        - xtrabackup_bin/implementation_details.md
+        - Connections and privileges:
+          - using_xtrabackup/privileges.md
+          - Set user privileges and permissions: howtos/permissions.md
+        - using_xtrabackup/configuring.md
+        - using_xtrabackup/comparison.md
+      - installation.md
+      - Use APT:
+        - Install using APT repository: installation/apt_repo.md
+        - Files in DEB package: installation/apt-files.md
+        - Build APT packages: installation/build-apt-packages.md
+        - Install from downloaded DEB packages: installation/apt-download-deb.md
+        - Apt pinning: installation/apt-pinning.md
+        - Run Percona XtraBackup: installation/apt-run.md
+        - Uninstall: installation/apt-uninstall-xtrabackup.md
+      - Use YUM:
+        - Install using a RPM repository: installation/yum_repo.md
+        - Files in RPM package: installation/yum-files.md
+        - Install from downloaded RPM packages: installation/yum-download-rpm.md
+        - Run Percona XtraBackup: installation/yum-run-xtrabackup.md
+        - Uninstall: installation/yum-uninstall-xtrabackup.md
+      - Use binary tarballs:
+        - Install with binary tarballs: installation/binary-tarball.md
+        - Binary tarballs available: installation/binary-tarball-names.md
+      - Compile from source:
+        - installation/compiling_xtrabackup.md
+      - Docker:
+        - installation/docker.md
+      - Secure a backup:
```

Comment:
Change to 
Secure
These files do not secure a backup - they secure an installation

Reply:
Updated